# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/ViewportUtils.cpp
# Commit: 071e6527d743
# Full Hash: 071e6527d74348ac8940e60018acc76bd7b44714
# Author: Botond Ballo <botond@mozilla.com>
# Date: 2020-05-06 09:37:16
# Regressor Bug: 1556556
# File Overlap Count: 1
# Description:
#   Bug 1556556 - Rename GetCallbackTransform() to GetVisualToLayoutTransform(). r=kats
#   
#   We will be applying and unapplying this transform in many places, and
#   thinking about those operations as "applying the callback transform" and
#   "unapplying the callback transform" is not very intuitive, especially since
# ==============================================================================

diff -r bda050135c64 -r 071e6527d743 layout/base/ViewportUtils.cpp
--- a/layout/base/ViewportUtils.cpp	Tue May 05 19:22:12 2020 +0000
+++ b/layout/base/ViewportUtils.cpp	Tue May 05 19:37:26 2020 +0000
@@ -14,7 +14,7 @@
 using layers::APZCCallbackHelper;
 using layers::ScrollableLayerGuid;
 
-CSSToCSSMatrix4x4 ViewportUtils::GetCallbackTransform(
+CSSToCSSMatrix4x4 ViewportUtils::GetVisualToLayoutTransform(
     ScrollableLayerGuid::ViewID aScrollId) {
   if (aScrollId == ScrollableLayerGuid::NULL_SCROLL_ID) {
     return {};